plat/allwinner/common/sunxi_common.c

```diff
@@ -54,3 +55,19 @@ void sunxi_configure_mmu_el3(int flags)
 
 	enable_mmu_el3(0);
 }
+
+#define SRAM_VER_REG (SUNXI_SYSCON_BASE + 0x24)
```

Comment:
It was there (see 1f4b638), but looked weird there as well ;-) The thing is that this is really an offset into this "syscon" device, and thus device related, not memmap related.
The idea is that all device base addresses are in sunxi_mmap.h, in a per-target directory. This allows SoCs with a different memory map, but the same devices to blend in nicely (see 61eedb07963e). I made [one exception](../blob/master/plat/allwinner/sun50i_a64/include/sunxi_mmap.h#L39-L40), because the clock devices are different between SoCs, but for ATF's purposes it's not worth to model this completely. So this one line saves tons of code.

Reply:
Ok, fair enough.